rules/okta_rules/okta_password_accessed.py

```diff
@@ -0,0 +1,39 @@
+from panther_base_helpers import deep_get, lst_iter
+
+
+def rule(event):
+    global target_user
+    global target_app_names
+    # event['target'] = [{...}, {...}, {...}]
+    target_user = lst_iter(
```

Comment:
if `target_user` is going to have a list for a value, it probably should be pluralized `target_users`